### PR TITLE
Fix wrong use of evm account address for assethub offramp

### LINKED ISSUE #326: Fix wrong use of evm account address for AssetHub offramp
## Context 

There currently are some flaws in the code that make the offramping flow assume and use the address of a connected EVM account instead of the Substrate account. We should address these flaws. One symptom of this is that the 'Confirm' button does not do anything if no EVM account is connected.

## TODO

Change the code so that connecting a Substrate account is enough to start the flow.

### PR DESCRIPTION
**Pull request by ebma**
- [ ] Adjust SIWE for Substrate accounts

Closes #326.

### CODE REVIEW

**Review by gianfra-t — APPROVED** [review]
Looks good 👍.

File: src/hooks/useVortexAccount.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  const chainId = useMemo(() => {
+    if (selectedNetwork === Networks.AssetHub) {
+      return AssetHubChainId;
+    } else {
+      return evmChainId;
+    }
+  }, [selectedNetwork, evmChainId]);
+
+  const type = useMemo(() => {
```
Comment (Sharqiewicz) [inline src/hooks/useVortexAccount.ts:42]: I think we don't have to useMemo for variables here, there is no heavy computation that would require it, wdyt?

File: src/pages/progress/index.tsx [hunk 2]
```diff
@@ -180,7 +183,12 @@ export const ProgressPage: FC<ProgressPageProps> = ({ offrampingState }) => {
     <BaseLayout
       main={
         <main>
-          <ProgressContent currentPhase={currentPhase} currentPhaseIndex={currentPhaseIndex} message={message} />
+          <ProgressContent
+            currentPhase={currentPhase}
+            currentPhaseIndex={currentPhaseIndex}
+            message={message}
+            network={selectedNetwork}
```
Comment (Sharqiewicz) [inline src/pages/progress/index.tsx:190]: we don't have to pass `selectedNetwork` to this component. We can just use `useNetwork` inside of ProgressContent to reduce props drilling

File: src/hooks/useVortexAccount.ts (hunk 1 above)
Comment (ebma) [reply src/hooks/useVortexAccount.ts:42]: That's true but I decided to use it anyway to mitigate any potential future issues in other components if we decide to put the return values of this hook into the dependencies of another hook. We had those issues in the past and it was quite hard to debug IMO.

File: src/pages/progress/index.tsx (hunk 2 above)
Comment (ebma) [reply src/pages/progress/index.tsx]: Changed.

**Pull request merged**